quadproduction/OpenPype
# #398: enhancement: set fps to imported files when using loader (After Effects)

## Changelog Description
Retrieve FPS from asset data (or project data if not found) and set imported files through loader with retrieved value.

## Testing notes:
1. Open any After Effect scene
2. Load any file
3. Check that imported file framerate is the same that the one for the projet or the asset
